lib/chef/resource/powershell_script.rb

```diff
@@ -46,6 +44,8 @@ def convert_boolean_return(arg=nil)
       def self.get_default_attributes(opts)
         {:convert_boolean_return => true}
       end
+
+      private_class_method :get_default_attributes
```

Comment:
This was protected before, which means that subclasses are allowed to call it as a class method. Does that actually matter in this case? Are there subclasses of powershell script?